telethon/_client/messages.py

```diff
@@ -738,3 +738,41 @@ async def _pin(self, entity, message, *, unpin, notify=False, pm_oneside=False):
 
     # Pinning a message that doesn't exist would RPC-error earlier
     return self._get_response_message(request, result, entity)
+
+async def send_reaction(
+    self: 'TelegramClient',
+    entity: 'hints.EntityLike',
+    message: 'hints.MessageIDLike',
+    reaction: typing.Optional[str] = None
+):
+    message = utils.get_message_id(message) or 0
+    if not reaction:
+        get_default_request = _tl.fn.help.GetAppConfig()
```

Comment:
While this request is probably cheap, it's unfortunate it must be called so often. However, it's probably fine, for the time being.

Reply:
should the friendly method have mandatory emoji ? 

i guess, by removing the None .

Reply:
No, I'm fine with allowing a default reaction.